saas/axops/src/ui/src/app/common/multiple-service-launch-panel/multiple-service-launch-panel.component.ts

```diff
@@ -376,9 +376,9 @@ export class MultipleServiceLaunchPanelComponent {
         return this.session[parameterValue.substring(parameterValue.indexOf('.') + 1).toString()] || vTemp;
     }
 
-    private getFallbackTemplateValue(prop: string) {
-        if (this.templatesToSubmit.length === 1 && this.templatesToSubmit[0][prop]) {
-            return this.templatesToSubmit[0][prop];
+    private getFallbackTemplateValue(prop: string, index = 0) {
+        if (this.templatesToSubmit.length === 1 && this.templatesToSubmit[index][prop]) {
```

Comment:
Check `this.templatesToSubmit.length === 1` is not required any more.

Reply:
Fixed. Since the template is passed instead of the index, I deleted this helper function and just put the fallback value inline.